diff --git a/node_modules/prettier-plugin-package/lib/index.js b/node_modules/prettier-plugin-package/lib/index.js
index d424c7e..16aeaf0 100644
--- a/node_modules/prettier-plugin-package/lib/index.js
+++ b/node_modules/prettier-plugin-package/lib/index.js
@@ -8,7 +8,7 @@
   The above copyright notice and this permission notice shall be
   included in all copies or substantial portions of this Source Code Form.
 */
-const { parsers } = require('prettier/parser-babylon');
+const { parsers } = require('prettier/parser-babel');
 
 const { engines } = require('./rules/engines');
 const { files } = require('./rules/files');
